app/components/reports/reports-filters.tsx

```diff
@@ -0,0 +1,144 @@
+import { Filter } from "lucide-react";
+import { Button } from "~/components/ui/button";
+import { Checkbox } from "~/components/ui/checkbox";
+import {
+	Dialog,
+	DialogClose,
+	DialogContent,
+	DialogFooter,
+	DialogTrigger,
+} from "~/components/ui/dialog";
+import {
+	Drawer,
+	DrawerClose,
+	DrawerContent,
+	DrawerFooter,
+	DrawerTrigger,
+} from "~/components/ui/drawer";
+import { Separator } from "~/components/ui/separator";
+import { Slider } from "~/components/ui/slider";
+
+interface ReportFiltersProps {
+	outlets: string[];
+	states: string[];
+	amounts: number[];
+}
+
+const FilterItems: React.FC<ReportFiltersProps> = ({
+	outlets,
+	states,
+	amounts,
+}) => {
+	const minAmountNeeded = Math.min(...amounts);
+	const maxAmountNeeded = Math.max(...amounts);
+	const rangeBetweenAmounts = maxAmountNeeded - minAmountNeeded;
+	return (
+		<div className="p-6">
+			<h2 className="pl-4 text-lg font-medium">Amount needed</h2>
+			<div className="px-6 py-8">
+				<Slider
+					defaultValue={[
+						Math.floor(rangeBetweenAmounts * 0.25),
+						Math.floor(rangeBetweenAmounts * 0.75),
+					]}
+					min={minAmountNeeded}
+					max={maxAmountNeeded}
+					step={4}
+					minStepsBetweenThumbs={18}
+				/>
+			</div>
+			<Separator className="bg-vd-blue-500 my-6 md:my-10" />
+			<div className="flex flex-col md:flex-row md:justify-between">
+				<div className="pl-4">
+					<h2 className="text-lg font-medium pb-2 md:pb-4">Story from</h2>
+					{outlets.map((outlet: string) => (
+						<div key={outlet} className="flex items-center gap-2 pb-2">
+							<Checkbox className="h-6 w-6 rounded-full border-vd-blue-500 data-[state=checked]:bg-vd-blue-500 data-[state=checked]:text-vd-beige-100" />
+							<p className="text-base">{outlet}</p>
+						</div>
+					))}
+				</div>
+				<Separator
+					className="md:hidden bg-vd-blue-500 my-6"
+					orientation="horizontal"
+				/>
+				<Separator
+					className="hidden md:flex bg-vd-blue-500 mx-6"
+					orientation="vertical"
```

Comment:
can anyone spot what's wrong here? I haven't been able to get a vertical separator to show up, with this component or with a `<div classname="w-[1px] h-full"/>`

Reply:
Wrap both in a flex div and it should work. 

I think the design should account for more than five options and selecting multiple at once. This component is usually called a "Multiselect". Please investigate!

Reply:
> I think the design should account for more than five options and selecting multiple at once.

not sure what you mean by more than 5 options.. are you referring to the longer list of states as reports are added?

Reply:
> > I think the design should account for more than five options and selecting multiple at once.
> 
> not sure what you mean by more than 5 options.. are you referring to the longer list of states as reports are added?

yes, that general rule applies to any other input types that apply.

- less than 5 options: radio / checkbox
- 5 or more options: dropdown <-- VD will most likely fit here